client/utils/featureFlags.ts

```diff
@@ -1,6 +0,0 @@
-import { getClientInitialData } from './initialData';
```

Comment:
So it turns out that this doesn't work at all because `getClientInitialData()` doesn't work during the SSR stage. I think there's a more careful version of this that we could write, but for now we'll just get feature flags from `usePageContext()` — I updated the docs to reflect this.